build.gradle

```diff
@@ -1,13 +1,12 @@
 buildscript {
     repositories {
-        mavenLocal()
         maven { url "https://oss.jfrog.org/oss-snapshot-local" }
```

Comment:
I believe this snapshot repo can be removed as well?

Reply:
Yeah, thought about removing it but wasn't sure if it was there because of something else.

I'll get rid of it and see what happens.